flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/StateAssignmentOperationTest.java

```diff
@@ -563,7 +563,7 @@ private InflightDataGateOrPartitionRescalingDescriptor gate(
     }
 
     @Test
-    void testChannelStateAssignmentTwoGatesPartiallyDownscaling()
+    public void testChannelStateAssignmentTwoGatesPartiallyDownscaling()
```

Comment:
NIT: With Junit 5, methods are not required to be public.

Reply:
Ahhh, that explains my confusion when I was cherry-picking some things  between branches. Anyway, next time I will just skip this.